ajax/libs/spritejs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "spritejs",
+  "version": "3.2.8",
```

Comment:
```suggestion
```
Please do not include the version attr, the bot will automatically provide this when it pulls in files.